src/main.rs

```diff
@@ -105,6 +106,14 @@ async fn main() {
         }
     };
 
+    match crate::persistence::init_db().await {
```

Comment:
the monitor module used to initialize its own persistence, now the top level persistence module initializes its own submodules (such as the `notification_logs` submodule) and all other nested persistence modules (such as the `monitor/persistence` module)

Reply:
understandable